### PR TITLE
documentation-typo

### PR DESCRIPTION
**Pull request by m7pr**

### CODE REVIEW

**Review by vedhav — APPROVED** [review]
LGTM!

**Comment by m7pr** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by m7pr** [comment]
recheck

File: R/choices_labeled.R [hunk 1]
```diff
@@ -166,7 +166,7 @@ choices_labeled <- function(choices, labels, subset = NULL, types = NULL) {
 #' variable_choices("ADRS", subset = c("USUBJID", "STUDYID"))
 #'
 #' # functional subset (with delayed data) - return only factor variables
-#' variable_choices("ADRS", subset = function(data) {
+#' variable_choices(ADRS, subset = function(data) {
```
Comment (gogonzo) [inline R/choices_labeled.R:169]: This is not a "delayed" variable_choices when `ADRS` is passed directly.

**Pull request merged**